client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -20,11 +62,7 @@ const ReadingSettings = () => {
 		<Main className="site-settings">
```

Comment:
We could consider adding the `settings-reading` class (similarly as we have for the Writing settings section already):

```suggestion
		<Main className="settings-reading site-settings">
```

Reply:
Good point. Something that needs to be looked at. I wonder if the classes are needed at all. 

Why would we need the `settings-reading` class? I know you said that for example the Writing settings do this. I noticed that too. However I did not include it and did not see any side effects. If there is no use for it, then we should not include it.

Reply:
I double checked now and I did not find a valid reason for setting `settings-reading` classname on this component and similar other ones that you highlighted.
![Screen Shot 2022-12-08 at 11 37 03](https://user-images.githubusercontent.com/2019970/206426602-95299658-0e8a-4af7-96e4-6afbd6e50c83.png)
![Screen Shot 2022-12-08 at 11 37 14](https://user-images.githubusercontent.com/2019970/206426610-46cc3c8e-d058-4b9b-8750-0d8f06fc8c94.png)
![Screen Shot 2022-12-08 at 11 37 33](https://user-images.githubusercontent.com/2019970/206426619-aa8b31a0-4cd8-4747-8e0e-c686c4678e78.png)
![Screen Shot 2022-12-08 at 11 39 44](https://user-images.githubusercontent.com/2019970/206426625-f4a3fad0-ace7-42e0-baed-5feed8c1a1ec.png)

Reply:
Thanks for checking that, Maciej. My thinking behind the consideration was to keep the code consistent with other setting sections. But that's a good point - since those classes are not used, we can skip them.

Reply:
Likely to make writing tests easier.

Reply:
> Likely to make writing tests easier.

Good to know!